src/Symfony/Component/Security/Core/Authentication/RememberMe/TokenProviderInterface.php

```diff
@@ -39,7 +39,7 @@ public function deleteTokenBySeries(string $series);
      *
      * @throws TokenNotFoundException if the token is not found
      */
-    public function updateToken(string $series, #[\SensitiveParameter] string $tokenValue, \DateTime $lastUsed);
+    public function updateToken(string $series, #[\SensitiveParameter] string $tokenValue, \DateTimeInterface $lastUsed);
```

Comment:
This is a BC break for implementors of the interface

Reply:
Can we remove the param from 6.2 and replace it with an `@param`annotation? I think you said we cannot but I forgot the reasoning once again :grimacing:

Reply:
It doesn't have a default value, which means removing the param is not allowed as that would mean the implementors are flagged as incompatible with the interface?